chart/Chart.yaml

```diff
@@ -31,7 +31,7 @@ sources:
 - https://github.com/dNationCloud/kubernetes-monitoring-stack
 dependencies:
 - name: kube-prometheus-stack
-  version: "30.0.*"
+  version: "39.5.*"
```

Comment:
It seems to work but can we try 39.6.*, most recent one ?

Reply:
`kube-prometheus-stack` version has been updated to the latest minor: `"39.6.*"`